game.cpp

```diff
@@ -1134,6 +1134,25 @@ void test_game() //!OCLINT tests may be many
   }
 #endif
 
+//#define FIX_ISSUE_463
+#ifdef FIX_ISSUE_463
+  // Players lose invulnerability after a short time
+  {
+    game g;
+    const int duration_invulnerability = 3;
```

Comment:
This constant should in the future be part of `game`/`game_options` class. This is a good first step, with helpful comments.

Reply:
Seems like a nice thing to create an Issue for!

Reply:
Agreed! Implementing this member variable is actually an implicit step of the issue. But I guess issues shouldn't have implicit steps!
I'd like to push the tests first in case someone wants to tackle it today, but I'll rework it to implement this if nobody takes it